### PR TITLE
Bump Operator to 1.14

### PR DESCRIPTION
**Pull request by creydr**
Bump operator to 1.14.1 and update patches

### CODE REVIEW

**Comment by creydr** [comment]
@skonto @pierDipi could you PTAL?

File: olm-catalog/serverless-operator/manifests/operator_v1beta1_knativeserving_crd.yaml [hunk 1]
```diff
@@ -2352,5 +2369,5 @@ spec:
       clientConfig:
         service:
           name: operator-webhook
-          namespace: default
+          namespace: knative-operator
```
Comment (pierDipi) [inline olm-catalog/serverless-operator/manifests/operator_v1beta1_knativeserving_crd.yaml:2372]: Is this change correct for us?

File: olm-catalog/serverless-operator/manifests/operator_v1beta1_knativeeventing_crd.yaml [hunk 2]
```diff
@@ -2272,5 +2278,5 @@ spec:
       clientConfig:
         service:
           name: operator-webhook
-          namespace: default
+          namespace: knative-operator
```
Comment (pierDipi) [inline olm-catalog/serverless-operator/manifests/operator_v1beta1_knativeeventing_crd.yaml:2281]: same question

File: olm-catalog/serverless-operator/manifests/operator_v1beta1_knativeserving_crd.yaml (hunk 1 above)
Comment (ReToCode) [reply olm-catalog/serverless-operator/manifests/operator_v1beta1_knativeserving_crd.yaml:2372]: Hm needs to be tested. IMHO we rewrite all namespaces to our own ones anyway. So probably yes?

Comment (pierDipi) [reply olm-catalog/serverless-operator/manifests/operator_v1beta1_knativeserving_crd.yaml:2372]: it's managed by OLM ? https://github.com/openshift-knative/serverless-operator/blob/473a9c576ba29bea28c653791e9541798e9d9dfc/olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml#L1283-L1304

**Pull request merged**